deploy/olm-catalog/performance-addon-operator/4.8.0/performance-addon-operator.v4.8.0.clusterserviceversion.yaml

```diff
@@ -119,7 +119,8 @@ metadata:
       ]
     capabilities: Basic Install
```

Comment:
please ignore changes on clusterserviceversion file and merge changes to `go.mod` and `go.sum` in the previous commit.